test/fast.js

```diff
@@ -157,6 +157,46 @@ describe('aws4', function() {
     })
   })
 
+  describe('#uriEscape()', function() {
+    var uriEscape = function (str) {
```

Comment:
Yeah, I know, that was stupid but I was in a hurry to commit this. It's the same function, copy-pasted. Sorry about that!